### PR TITLE
Update gradle version and other deps

### PR DESCRIPTION
**Pull request by fstanis**
This allows the project to build with the latest Android Studio.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,12 +41,12 @@ task clean(type: Delete) {
 }
 
 ext {
-    buildToolsVersion = "26.0.1"
-    supportLibVersion = "26.0.2"
+    buildToolsVersion = "28.0.3"
+    supportLibVersion = "26.1.0"
```
Comment (benbaxter) [inline build.gradle:45]: It would be ideal to update to 28.

**Review by benbaxter — APPROVED** [review]
Can you update the target/compile sdk as well?

File: build.gradle (hunk 1 above)
Comment (fstanis) [reply build.gradle:45]: Ack - will try to do a more thorough update in the coming days.

Comment (tjohns) [reply build.gradle:45]: Resolved in commit c2457e6

**Pull request merged**